packages/dd-trace/src/config.js

```diff
@@ -19,6 +19,7 @@ const telemetryMetrics = require('./telemetry/metrics')
 const { getIsGCPFunction, getIsAzureFunction } = require('./serverless')
 const { ORIGIN_KEY, GRPC_CLIENT_ERROR_STATUSES, GRPC_SERVER_ERROR_STATUSES } = require('./constants')
 const { appendRules } = require('./payload-tagging/config')
+const libdatadog = require('@datadog/libdatadog')
```

Comment:
This should not always be loaded as it won't be available in serverless environments. This probably means we need to either revisit how libdatadog is packaged or how unwanted files can be excluded, but in the meantime it should be loaded only when needed and when available.

See https://github.com/DataDog/datadog-lambda-js/blob/9e4af128689ec9832f491e676a487f5f9fa99d6d/Dockerfile#L37

Reply:
It could work to some extent on serverless, but it's not in the current scope. If it's possible I'll add conditional loading to avoid serverless.

Reply:
Done in [address review](https://github.com/DataDog/dd-trace-js/pull/5126/commits/4e83b16fa4e1e31c7f3ef42c9e4f71f13287c12f)